api/app/jobs/rdps_sfms.py

```diff
@@ -0,0 +1,177 @@
+"""
+A script that downloads RDPS temp, rh, precip and wind speed weather model data from the Environment Canada HTTP data server.
+Data is stored in S3 storage for a maximum of 7 days
+"""
+
+import asyncio
+import os
+import sys
+from datetime import datetime
+from collections.abc import Generator
+import logging
+import tempfile
+from sqlalchemy.orm import Session
+from app.db.database import get_write_session_scope
+from app.db.crud.weather_models import create_model_run_for_sfms, create_saved_model_run_for_sfms_url, get_saved_model_run_for_sfms
+from app.jobs.common_model_fetchers import CompletedWithSomeExceptions, download
+from app.weather_models import ModelEnum, ProjectionEnum
+from app import configure_logging
+import app.utils.time as time_utils
+from app.utils.s3 import get_client
+from app.rocketchat_notifications import send_rocketchat_notification
+from app.jobs.env_canada_utils import get_regional_model_run_download_urls
+
+# If running as its own process, configure logging appropriately.
+if __name__ == "__main__":
+    configure_logging()
+
+logger = logging.getLogger(__name__)
+
+
+DAYS_TO_RETAIN = 7
+MAX_MODEL_RUN_HOUR = 37
+GRIB_LAYERS = {"temp": "TMP_TGL_2", "rh": "RH_TGL_2", "precip": "APCP_SFC_0", "wind_speed": "WIND_TGL_10"}
+
+
+def get_model_run_hours_to_process() -> Generator[int, None, None]:
+    """Yield the 00 and 12 model run hours for RDPS (skip 06 and 18 for now)"""
+    for hour in [0, 12]:
+        yield hour
+
+
+class RdpsGrib:
```

Comment:
Minor: Wonder if we should stick to uppercase for the acronym names

Reply:
Do you mean the keys in the `GRIB_LAYERS` dictionary?

Reply:
Oh I just mean renaming the class to be `RDPSGrib`